### PR TITLE
Remove tools/quic and aioquic

### PR DESCRIPTION
**Pull request by bashi**
For WebTransport over HTTP/3, we plan to use virtualenv and
requirements.txt for aioquic dependency.

### CODE REVIEW

**Comment by bashi** [comment]
@foolip PTAL?

cc/ @yutakahirano @nidhijaju

File: tools/wptrunner/wptrunner/environment.py [hunk 1]
```diff
@@ -274,7 +271,7 @@ def test_servers(self):
 
         if not failed and self.test_server_port:
             for scheme, servers in self.servers.items():
-                if scheme == "quic-transport" or scheme == "webtransport-h3":
+                if scheme == "webtransport-h3":
                     # TODO(bashi): Find a way to test QUIC's UDP port.
```
Comment (foolip) [inline tools/wptrunner/wptrunner/environment.py:275]: Should this comment change somehow?

Comment (bashi) [reply tools/wptrunner/wptrunner/environment.py]: Updated.

**Comment by bashi** [comment]
Thank you for review and suggestions, @foolip!

**Review by yutakahirano — APPROVED** [review]

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@jgraham your review is needed because this touches tools/third_party/.

**Review by jgraham — APPROVED** [review]

**Comment by bashi** [comment]
Thank you for review everyone!

**Pull request merged**